stanford/roles/jenkins_analytics_configs/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Create additional configuration files
+  template:
+    src: "{{ item }}.j2"
+    dest: "{{ jenkins_home }}/{{ item }}"
+    owner: "{{ jenkins_user }}"
+    group: "{{ jenkins_group }}"
+    mode: "0400"
```

Comment:
Should the group (`jenkins`) not also have at least read permissions?
And should the owner (`jenkins`) have write permissions?

`mode: '0640'`